comps/llms/text-generation/predictionguard/docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2024 Prediction Guard, Inc.
```

Comment:
The content of this PR looks good to me. just a comment on the folder structure, we have refined it a bit to remove the 'docker' folder and flat all the files at a same parent folder. could you pls refer to the latest master and make corresponding changes?